docs/404.html

```
@@ -0,0 +1,212 @@
+                <ul class="footer-links">
+                    <li><a href="https://github.com/ikafly144/sabalauncher" target="_blank">GitHub</a></li>
+                    <li><a href="https://github.com/ikafly144/sabalauncher/releases" target="_blank">リリース</a></li>
+                    <li><a href="https://github.com/ikafly144/sabalauncher/issues" target="_blank">バグ報告</a></li>
+                </ul>
+            </div>
+        </div>
+
+        <div class="footer-bottom">
+            <p>&copy; 2025 ikafly144. All rights reserved.</p>
+        </div>
+        </div>
```

HTMLHint (1.5.0)

[error] Tag must be paired, no start tag: [ </div> ]

(tag-pair)